src/main/webapp/user.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Get the user @param user from the user servlet.
+ * @param {String} user
+ * @return a user object
+ */
+function getUser(user) {
+  console.log('entering get user function/n');
+  fetch('/user?user=' + user)
+    .then(errorHandling).then((response) => response.json())
+    .then((user) => {
+      return user;
+    })
+    .catch((error) => {
+      console.log(error);
+    });
```

Comment:
Move this over to the [standardized fetch error handler](https://github.com/googleinterns/step22-2020/blob/projects-preview-frontend/src/main/webapp/js/fetch_handler.js) in the future. Totally fine for now, especially because the error handler isn't even merged with master yet lol.